src/api/lib/tasks/dev.rake

```diff
@@ -170,6 +170,60 @@ namespace :dev do
     end
   end
 
+  # Run this task with: rails "dev:notifications:data[3]"
+  # replacing 3 with any number to indicate how many times you want this code to be executed.
+  namespace :notifications do
+    task :data, [:repetitions] => :environment do |_t, args|
+      unless Rails.env.development?
+        puts "You are running this rake task in #{Rails.env} environment."
+        puts 'Please only run this task with RAILS_ENV=development'
+        puts 'otherwise it will destroy your database data.'
+        return
+      end
+
+      args.with_defaults(repetitions: 1)
+      repetitions = args.repetitions.to_i
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      admin = User.where(login: 'Admin').first
+      home_admin = admin.home_project
+      unless home_admin
+        home_admin = create(:project, name: admin.home_project_name)
+        create(:relationship, project: home_admin, user: admin, role: Role.hashed['maintainer'])
+      end
+      requestor = User.where(login: 'Requestor').first || create(:confirmed_user, login: 'Requestor')
+      requestor_project = create(:project, name: "requestor_project_#{Time.now.to_i}")
+
+      repetitions.times do |repetition|
+        timestamp = Time.now.to_i
+        admin_package = create(:package_with_file, name: "admin_package_#{timestamp}_#{repetition}", project: home_admin, file_content: 'from admin home')
```

Comment:
Why not use faker? like `Faker::Lorem.word`

Reply:
Much better, updated!